content/source/docs/cloud/index.html.md

```diff
@@ -1,13 +1,17 @@
 ---
 layout: "cloud"
-page_title: "Home - Terraform Cloud"
+page_title: "Home - Terraform Cloud and Terraform Enterprise"
 description: |-
   Terraform Cloud is an application that helps teams use Terraform to provision infrastructure.
 ---
 
-# Terraform Cloud
+# Terraform Cloud and Terraform Enterprise
 
-This is the documentation for Terraform Cloud.
+This is the documentation for Terraform Cloud and Terraform Enterprise.
+
+Terraform Cloud and Terraform Enterprise are different distributions of the same application; this documentation applies to both of them except when we specifically state otherwise.
```

Comment:
Alternative wording to remove "we" since I'm not sure if the document is meant to be written in the first-person perspective. Feel free to ignore.

```suggestion
Terraform Cloud and Terraform Enterprise are different distributions of the same application; this documentation applies to both of them except when specifically stated otherwise.
```